lib/puppet/parser/functions/parse_auto_update_option.rb

```diff
@@ -1,6 +1,15 @@
+#
+# parse_auto_update_option.rb
```

Comment:
Why has this been added?

Reply:
I've changed the comment to have 100.00% documentation. 
Files:                    6
Modules:                  1 (    0 undocumented)
Classes:                  0 (    0 undocumented)
Constants:                0 (    0 undocumented)
Attributes:               0 (    0 undocumented)
Methods:                  0 (    0 undocumented)
Puppet Classes:           1 (    0 undocumented)
Puppet Defined Types:     1 (    0 undocumented)
Puppet Types:             0 (    0 undocumented)
Puppet Providers:         0 (    0 undocumented)
Puppet Functions:         3 (    0 undocumented)
Puppet Tasks:             1 (    0 undocumented)
Puppet Plans:             0 (    0 undocumented)
 100.00% documented

If they are useless we can remove them :)